docs/src/installation_instructions.md

```diff
@@ -6,15 +6,16 @@ Oceananigans. If you're new to Julia, we also recommend reading [Julia's "gettin
 
 ## Installation instructions
 
-The hardest part of installing Oceananigans is actually installing Julia (and even that should be pretty easy). 
-If you want to use Oceananigans on a laptop you can generally install Julia easily via the package manager of whatever
-operational system you use (some quick googling should tell you how). If you're running on a cluster, you should check
-if it already has Julia installed (most systems these days do). If that's the case, then the hardest part is already done!
-
-If those options above are not possible, one alternative is to simply download the Julia pre-compiled binaries from the
-[Julia website](https://julialang.org/downloads/), where you can simply run the Julia executable that's contained inside
-(this should work out of the box!). Another option is to compile Julia manually from source code. This is definitely the hardest
-option, and you can find instructions on how to do that [here](https://github.com/JuliaLang/julia#building-julia).
+Usually the hardest part of installing Oceananigans is installing Julia. The main options here in order
+of simple to complex are
+
+1. Check if your system already has Julia installed. Most HPC systems these days have Julia
+   available.
+2. Install Julia via the package manager of the operational system you use. E.g. `apt` for Ubuntu.
```

Comment:
I don't think Julia recommends this, so we shouldn't, right? I don't see it mentioned here:

https://julialang.org/downloads/platform/#linux_and_freebsd

There is however an installer for windows:

https://julialang.org/downloads/platform/#juliaup